File: OrderNet-Server/src/main/java/bitc/fullstack503/ordernetserver/dto/app/BranchCountDTO.java
package bitc.fullstack503.ordernetserver.dto.app;

import lombok.Data;

@Data
public class BranchCountDTO {

    // 대리점 명
    private String branchName;

    // 주문 수 - 승인대기
    private int orderCount;

    // 접수 된 주문 수 - 승인 완료, 출고 대기
    private int acceptCount;

    // 출고 된 주문 수 - 출고
    private int deliveryCount;

    // 반려 된 주문 수
    private int holdCount;
}

File: OrderNet-APP/app/src/main/java/bitc/fullstack/app/dto/BranchCountDTO.kt
package bitc.fullstack.app.dto

data class BranchCountDTO(

    // 대리점 명
    val branchName: String,

    // 주문 수 - 승인대기
    val orderCount: Int,

    // 접수 된 주문 수 - 승인 완료, 출고 대기
    val acceptCount: Int,

    // 출고 된 주문 수 - 출고
    val deliveryCount: Int,

    // 반려 된 주문 수
    val holdCount: Int
)
